conf/nutch-default.xml

```diff
@@ -1896,4 +1896,33 @@ CAUTION: Set the parser.timeout to -1 or a bigger value than 30, when using this
   </description>
 </property>
 
+<!-- index-links plugin -->
```

Comment:
Any thoughs on a good prefix to use? `index.links.outlinks.host.ignore` seems extremely large for my taste :)